indy_node/test/upgrade/test_forced_upgrade_if_ordered_and_then_request_received.py

```diff
@@ -32,6 +32,6 @@ def test_forced_upgrade_handled_once_if_ordered_and_then_request_received(
     looper.runFor(waits.expectedUpgradeScheduled())
 
     checkUpgradeScheduled([slow_node], validUpgradeExpForceTrue[VERSION])
-    assert len(list(slow_node.upgrader._actionLog)) == 1
-    assert slow_node.upgrader._actionLog.lastEvent[1] == \
-           UpgradeLog.SCHEDULED
+    assert count_action_log_entries(list(slow_node.upgrader._actionLog),
```

Comment:
It looks like we use the same lambda in all (or most) of the tests. Maybe move it to a helper method (or extend `count_action_log_entries`)?